test/.eslintrc.json

```diff
@@ -11,6 +11,7 @@
     "no-param-reassign": "off",
     "no-console": "off",
     "curly": "off",
+    "quotes": ["error", "single", "avoid-escape"],
```

Comment:
Let me know if you'd like to promote this to the top-level config.

Reply:
Are there any escaped quotes outside of `test/`? I can't find any in `src/`.

Reply:
Guess there's not. Let's keep it here then.